manage.py

```diff
@@ -177,21 +177,21 @@ def truncate_recording_artist_join_table():
 
 
 @cli.command()
-@click.option("--verbose", "-v", default=0, help="Print debug information for given verbose level(0,1,2).")
-def create_artist_credit_clusters_for_mbids(verbose=0):
+@click.option("--verbose", "-v", default='WARNING', help="Print debug information for given verbose level(WARNING, INFO, DEBUG).")
+def create_artist_credit_clusters_for_mbids(verbose='WARNING'):
     """Creates clusters for artist_credits using artist MBIDs present in
        recording_json table.
     """
 
-    if verbose == 1:
-        logging.basicConfig(format='%(message)s', level=logging.INFO)
-    elif verbose == 2:
-        logging.basicConfig(format='%(message)s', level=logging.DEBUG)
+    try:
+        if verbose == 'INFO':
+            logging.basicConfig(format='%(message)s', level=logging.INFO)
+        elif verbose == 'DEBUG':
+            logging.basicConfig(format='%(message)s', level=logging.DEBUG)
```

Comment:
What happens when the user passes "DOOFUS" from the command line? You should check to make sure that passed values are always legal. If no valid ones are passed, you might give a warning that it was ignored and carry on using the default logging value.